docs-source/spring/content/development/cli/_index.md

```diff
@@ -560,3 +560,174 @@ The following is an example development workflow using the CLI:
          oractl:>config list --service-name myserv --service-profile obaas --service-label 0.1 --property-key ktest2
          400 : "Couldn't find any property for submitted query."
          ```
+8. Use the `GraalVM Compile Commands` to upload a **.jar** file to the Oracle Backend for Spring Boot and Microservices and its GraalVM compiler service, to start a compilation of your microservice to produce an executable native file **.exec**, to get last logs available regarding a compilation in progress or terminated, to download the **.exec** file to deploy on the backend, or to purge files remaining after a compilation on remote GraalVM compiler service.
```

Comment:
If possible, turn this into a bulleted list.

Reply:
ok